frontend/src/metabase/modes/components/drill/ActionClickDrill/ActionClickDrill.tsx

```diff
@@ -1,62 +1,29 @@
 import { getDataFromClicked } from "metabase/lib/click-behavior";
 
-import {
-  executeRowAction,
-  openActionParametersModal,
-} from "metabase/dashboard/actions";
+import { openActionParametersModal } from "metabase/dashboard/actions";
 
-import type { ParameterMappedForActionExecution } from "metabase-types/api";
 import type { ActionClickObject } from "./types";
 
-import { prepareParameter, getNotProvidedActionParameters } from "./utils";
-
 function ActionClickDrill({ clicked }: { clicked: ActionClickObject }) {
-  const { dashboard, dashcard } = clicked.extraData;
+  const { dashcard } = clicked.extraData;
+  const { onSubmit, missingParameters } = clicked;
```

Comment:
i just kind of shoved these params onto the clicked object? is there a more idiomatic place to put them?